indico/htdocs/sass/modules/_papers.scss

```diff
@@ -39,3 +39,23 @@
         }
     }
 }
+
+#assign-role-table {
```

Comment:
Writing down this only to trigger a little discussion. I think we should try to avoid as much as possible using IDs for styling. For several reasons.

It's true that we want to scope changes and ID selectors are a good way to do that. When we have no other choice but to that I think we should try to use the broadest scope possible that fit. In this case it would be the ID of the `page_class` defined in the template.

However, I think we should go even further and try not to develop so many module/page specific components and component tweaks. Instead, if we need a slight variation we should try to add a new "mode" in the component. The reason is re-usability and maintainability. On the one hand if we ever make changes to the component we clearly see all the other modes that may be affected without having to search the full project. On the other hand, if we ever need to re-use the slight tweaks, they are already abstracted and ready to use simply adding a class. 

One side effect of this approach is that it's also easier to see in the HTML if the original component is modified. In this case, for instance, we would use `<div class="i-tag i-tag--special">blablabla</div>`.

Thoughts? /cc @pferreir @ThiefMaster @nop33 @DavidAndreev 
(We can also discuss offline)